webstruct/tests/test_date_features.py

```diff
@@ -0,0 +1,109 @@
+from webstruct.features import (looks_like_day_ordinal,
+                                looks_like_date_pattern,
+                                number_looks_like_day,
+                                number_looks_like_month)
+
+
+class HtmlToken():
+    def __init__(self, token):
+        self.token = token
+
+
+def test_looks_like_day_ordinal():
+    token = HtmlToken('1st')
+    expected = {'looks_like_day_ordinal': True}
+    result = looks_like_day_ordinal(token)
+    assert result == expected
```

Comment:
would you mind cleaning it up a bit - e.g. you can make a function `assert_looks_like_day_ordinal("1st", True)` to reduce copy-paste and make code more clear